lib/pbench/server/api/auth.py

```diff
@@ -150,20 +90,22 @@ def verify_auth(auth_token):
         except jwt.ExpiredSignatureError:
             try:
                 ActiveTokens.delete(auth_token)
-            except Exception:
+            except Exception as e:
                 Auth.logger.error(
-                    "User attempted Pbench expired token but we could not delete the expired auth token from the database. token: '{}'",
+                    "User passed expired token but we could not delete the token from the database. token: {!r}: {}",
                     auth_token,
+                    str(e),
                 )
-                return None
             Auth.logger.warning(
-                "User attempted Pbench expired token '{}', Token deleted from the database and no longer tracked",
+                "User passed expired token {!r}; token deleted from the database and no longer tracked",
```

Comment:
I realize this is not a change you made, but ... given this is a log message for consumption by the people administering the pbench server, is this really something to warn them about?  Seems like this is going to occur as a normal part of the user/server interactions.

Reply:
See also: https://github.com/distributed-system-analysis/pbench/discussions/2509